client/src/containers/explore/section/index.tsx

```diff
@@ -72,59 +46,64 @@ const Section: React.FC<PropsWithChildren<SectionProps>> = ({
   };
 
   return (
-    <section
-      className="h-[660px]"
-      ref={ref}
-      id={slug}
-      data-testid="section-container"
-    >
-      <header className="flex justify-between space-y-4 rounded-2xl bg-secondary p-6">
-        <div>
-          <h2 className="text-xl font-semibold">{name}</h2>
-          <p>{description}</p>
+    <section className="mb-16" id={slug} data-testid="section-container">
```

Comment:
Are we using `data-testid`?

Reply:
Also, I would add `grid` and `gap-0.5` classes so we make sure all elements inside the section are separated horizontally and vertically.

Reply:
We are using `section-container` in `explore.spec.ts`

Reply:
As a suggestion, I'd avoid using specific tags just for testing purposes whenever possible. I'd rather take advantage of the tool's codegen gen to locale elements by properties that are already existing, so we can avoid this.

I think it might be worth discussing it